install/helm/open-match/subcharts/open-match-test/templates/stress-service-account.yaml

```diff
@@ -0,0 +1,44 @@
+# Source: open-match/templates/podsecuritypolicy.yaml
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Breakout the workaround into a separate file since it'll be deleted soon.
Also I think this needs to be moved to open-match-core because enabling GKE metadata server will break Stackdriver integration. Stackdriver is using the node vm service account and if gke-metadata-server is broken it won't be able to get creds to write out.